src/sinks/aws_s_s/sns/config.rs

```diff
@@ -0,0 +1,125 @@
+use aws_sdk_sns::Client as SnsClient;
+
+use crate::config::{
+    AcknowledgementsConfig, DataType, GenerateConfig, Input, ProxyConfig, SinkConfig, SinkContext,
+};
+use vector_config::configurable_component;
+
+use super::{
+    client::SnsMessagePublisher, message_deduplication_id, message_group_id, BaseSSSinkConfig,
+    SSRequestBuilder, SSSink,
+};
+use crate::aws::create_client;
+use crate::aws::ClientBuilder;
+
+/// Configuration for the `aws_sns` sink.
+#[configurable_component(sink(
+    "aws_sns",
+    "Publish observability events to AWS Simple Queue Service topics."
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏

```suggestion
    "Publish observability events to AWS Simple Notification Service topics."
```